vision/realsense/bag_file.py

```diff
@@ -0,0 +1,112 @@
+"""
+The BagFile class is a child class of the camera, designed to be used for pre-recorded .bag files
+"""
+
+import cv2
+import numpy as np
+import pyrealsense2 as rs
+import template
```

Comment:
Will this work even if template is just the file name, and not a class itself? The other instance of from...import... that I'm using is importing a function from a class, so I thought I'd ask if it would work the same way.